src/core/lib/slice/slice_internal.h

```diff
@@ -36,7 +36,7 @@
 // Interned slices have specific fast-path operations for hashing. To inline
 // these operations, we need to forward declare them here.
 extern uint32_t grpc_static_metadata_hash_values[GRPC_STATIC_MDSTR_COUNT];
-extern uint32_t g_hash_seed;
+extern uint32_t grpc_core::g_hash_seed;
```

Comment:
Please write this as:

```
namespace grpc_core {
extern uint32_t g_hash_seed;
}  // namespace grpc_core
```

I'm actually a little surprised that the compiler is okay with this syntax.  But even if it is, I think the rewrite makes it easier to read.